### DISCUSSION TITLE
How to resize image

### DISCUSSION THREAD

**Discussion by venyyyyy** [warengonzaga/css-text-portrait-builder#12] [Q&A] [answered]
Hii Would like to ask on how to resize the image generated image text, like when I want to use portrait photos instead of landscape ones. Can I edit in the style sheets?

**Comment by warengonzaga** [comment, marked as answer]
Hi @venyyyyy, actually that's a good question. I'm currently thinking of adding it as an option in the config file.
For now, I would suggest editing the image manually or editing the ``base.scss`` file on `p` selector. 

I'll create an issue for this.
